f/php/bootstrap

```diff
@@ -25,15 +27,29 @@ $server->start(function ($type, $output) {
     }
 });
 
-// Wait for the server to start
-sleep(1);
-
 register_shutdown_function(function() use($server) {
     $server->stop();
 });
 
+$wait = 10000; // 10ms
+$timeout = 5000000; // 5 secs
+$elapsed = 0;
+
+do {
+    usleep($wait);
```

Comment:
why do we need this sleep here? the fsockopen() is a blocking operation and will return as soon as the socket is available, no?

Reply:
Hmm, it didn't in my test (on mac). It just returned directly. Does it work for you?

Reply:
hm maybe the socket will not block until the server side socket is available.

I did not test it

Reply:
did you check whether the script can find the server now faster then with a 1sec wait?

Reply:
I was also expecting guzzle to wait, but it disnt. Perhaps because it was refused directly.